framework/infrastructure/gcp/compute.py

```
@@ -558,6 +568,78 @@ def get_backend_service_backend_health(self, backend_service, backend):
             .execute()
         )
 
+    def create_serverless_neg(self, name: str, region: str, service_name: str):
+        """Creates a serverless NEG.
+
+        Args:
+            name: The name of the NEG.
+            region: The region in which to create the NEG.
+            service_name: The name of the Cloud Run service.
+            service_name format is "namespaces/{namespace}/services/{service}"
+
+        Returns:
+            The NEG selfLink URL
+        """
+        neg_body = {
+            "name": name,
+            "networkEndpointType": "SERVERLESS",
+            "cloudRun": {"service": service_name},
+        }
+
+        try:
+            logger.info("Creating serverless NEG %s in %s", name, region)
+            self.api.regionNetworkEndpointGroups().insert(
+                project=self.project, region=region, body=neg_body
+            ).execute()
+            neg = self.get_serverless_network_endpoint_group(name, region)
+            logger.info("Created serverless neg %s ", neg)
+            return neg
+
+        except Exception as e:
```

There's almost never a good reason to catch bare exceptions. Not that `Exception` masks errors like `TypeError` which is often is code typo, literally.

---

Removed.